pages/utilities/image-to-base64.tsx

```diff
@@ -100,9 +128,21 @@ export default function ImageToBase64() {
               readOnly
               className="mb-4 overflow-x-hidden"
             />
-            <Button variant="outline" onClick={() => handleCopyBase64(base64)}>
+            <Button
+              variant="outline"
+              onClick={() => handleCopyBase64(base64)}
+              disabled={isDisabled}
+            >
               {buttonBase64}
             </Button>
+            <Button
+              className="ml-2"
+              variant="outline"
+              onClick={() => setBase64("")}
+              disabled={isDisabled}
+            >
+              Clear
+            </Button>
```

Comment:
Great Idea!

let's wrap it with `div` with classes `flex justify-between` and make them look different, since they are 2 different actions

```
<div className="flex justify-between">
  <Button
    variant="outline"
    onClick={() => handleCopyBase64(base64)}
    disabled={isDisabled}
  >
    {buttonBase64}
  </Button>
  <Button
    className="ml-2"
    variant="default"
    onClick={() => setBase64("")}
    disabled={isDisabled}
  >
    Clear
  </Button>
</div>
```

<img width="585" alt="image" src="https://github.com/user-attachments/assets/61facaa9-1c52-4acb-ac09-aed416656674">